libs/base/ui/graphical/video/src/video.component.spec.ts

```diff
@@ -0,0 +1,209 @@
+import { fixture } from '@open-wc/testing-helpers';
+import { useComponent } from '@spryker-oryx/core/utilities';
+import { html } from 'lit';
+import VideoComponent from './video.component';
+import { videoComponent } from './video.def';
+
+describe('VideoComponent', () => {
```

Comment:
I believe that we need also test that changing of ascpectRatio prop sets the css properties